en/docs/administer/key-managers/configure-okta-connector.md

```diff
@@ -329,6 +329,9 @@ Follow the instructions below to configure Okta as a third-party Key Manager:
           </tr>
         </table>
 
+!!! tip
+     After the authorization URL, which is generated when you create an application in Okta, is provided as a **"Well-known-url"**, it can be found under the parameter **Metadata URI** in the settings related to the Authorization server. When you click the **Import** button under the latter mentioned settings, the required information in the Key Manager configurations will be filled automatically.
```

Comment:
Shall we simplify the tip?

``` suggestion
Key Manager configurations can be auto filled by clicking the import buttion after providing the well-known endpoint of Okta. The well-known URL can be found in Metadata URI, under **Settings** section 
```
We can attach the screenshot if required. WDYT?

Reply:
Yes, that seems better. Correct the spelling mistake near  'clicking the import buttion'